lib/pages/main/wallet_contents/settings/wallet_connect.dart

```diff
@@ -0,0 +1,257 @@
+import 'package:qubic_wallet/services/qubic_hub_service.dart';
+import 'package:qubic_wallet/services/wallet_connect_service.dart';
+import 'package:qubic_wallet/stores/application_store.dart';
+import 'package:qubic_wallet/stores/qubic_hub_store.dart';
+import 'package:qubic_wallet/stores/settings_store.dart';
+import 'package:qubic_wallet/styles/edge_insets.dart';
+import 'package:qubic_wallet/styles/text_styles.dart';
+import 'package:qubic_wallet/styles/themed_controls.dart';
+import 'package:settings_ui/settings_ui.dart';
+import 'package:walletconnect_flutter_v2/walletconnect_flutter_v2.dart';
+
+class WalletConnect extends StatefulWidget {
+  const WalletConnect({super.key});
+
+  @override
+  // ignore: library_private_types_in_public_api
+  _AboutWalletState createState() => _AboutWalletState();
+}
+
+class _AboutWalletState extends State<WalletConnect> {
+  final ApplicationStore appStore = getIt<ApplicationStore>();
+  final SettingsStore settingsStore = getIt<SettingsStore>();
+  final SecureStorage secureStorage = getIt<SecureStorage>();
+  final QubicHubStore qubicHubStore = getIt<QubicHubStore>();
+  final QubicHubService qubicService = getIt<QubicHubService>();
+  final GlobalSnackBar snackBar = getIt<GlobalSnackBar>();
+  final WalletConnectService walletConnectService =
+      getIt<WalletConnectService>();
+  bool isLoading = false;
+
+  TextEditingController pairController = TextEditingController();
+
+  bool walletConnectEnabled = false;
+
+  StreamSubscription<SessionConnect?>? sessionConnectSubscription;
+  StreamSubscription<SessionDelete?>? sessionDisconnectSubscription;
+  StreamSubscription<SessionProposalEvent?>? sessionProposalSubscription;
+  void setupWCEvents() {}
+
+  Map<String, SessionData> sessions = {};
+
+  @override
+  void dispose() {
+    super.dispose();
+    sessionConnectSubscription?.cancel();
+    sessionDisconnectSubscription?.cancel();
+    sessionProposalSubscription?.cancel();
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    setState(() {
+      walletConnectEnabled = settingsStore.settings.walletConnectEnabled;
+    });
+    if (mounted) {
+      if ((walletConnectEnabled) && (walletConnectService.web3Wallet != null)) {
+        walletConnectService.web3Wallet!
+            .getActiveSessions()
+            .forEach((key, value) {
+          setState(() {
+            sessions[key] = value;
+          });
+        });
+      }
+    }
+  }
+
+  List<String> getMethods(SessionData sessionData) {
+    final l10n = l10nOf(context);
+
+    List<String> methods = [];
+    if (sessionData.requiredNamespaces == null) {
+      return [];
+    }
+    if (sessionData.requiredNamespaces![Config.walletConnectChainId] == null) {
+      return [];
+    }
+    sessionData.requiredNamespaces?[Config.walletConnectChainId]!.methods
+        // ignore: avoid_function_literals_in_foreach_calls
+        .forEach((string) {
+      if ((string == "wallet_requestAccounts")) {
```

Comment:
please use the constants defined

Reply:
@sallymoc  Fixed and pushed to branch. Please check again and resolve